ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -88,10 +88,12 @@ message GetTransactionsRequest {
   string ledger_id = 1;
 
   // Beginning of the requested ledger section.
+  // This offset is exclusive: the response will not contain any transaction whose offset is less than or equal to this.
   // Required
   LedgerOffset begin = 2;
 
   // End of the requested ledger section.
+  // This offset is inclusive: the response will not contain any transaction whose offset is strictly greater than this.
```

Comment:
Maybe `will only contain transactions whose offset is less than or equal to this`?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8145/commits/bb8621be35dd50afa01c52909147198a72e84fed